src/resolver/resolve_path.zig

```diff
@@ -435,24 +459,6 @@ pub fn relativeNormalized(from: []const u8, to: []const u8, comptime platform: P
     return relativeToCommonPath(common_path, from, to, &relative_to_common_path_buf, always_copy, platform);
 }
 
-pub fn dirname(str: []const u8, comptime platform: Platform) []const u8 {
-    switch (comptime platform.resolve()) {
-        .loose => {
-            const separator = lastIndexOfSeparatorLoose(str) orelse return "";
-            return str[0..separator];
-        },
-        .posix => {
-            const separator = lastIndexOfSeparatorPosix(str) orelse return "";
-            return str[0..separator];
-        },
-        .windows => {
-            const separator = lastIndexOfSeparatorWindows(str) orelse return std.fs.path.diskDesignatorWindows(str);
-            return str[0..separator];
-        },
-        else => @compileError("unreachable"),
-    }
-}
-
```

Comment:
☝️ Deleted this `dirname` variation in favor of `std.fs.path.dirname` as this was very bare bones and unlike other variants.